packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -55,6 +62,7 @@ TimestampContainer.defaultProps = {
   padding: true,
   prefix: null,
   suffix: null,
+  locale: null,
```

Comment:
what if the locale is null (if we don't pass it in), does it fail nicely without breaking the component or should we set the default to like `en-gb`?

Reply:
Should do, due to the `if (locale) {`

Reply:
if the locale is null, we don't set it. I think that is handled with
```js
if(locale) {
}
```
Moment defaults to `en` if no locale is defined.